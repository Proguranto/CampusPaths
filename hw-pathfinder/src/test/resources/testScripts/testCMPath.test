# Find a path with Dijkstra's algorithm when
# the graph is cyclic and there is multiple paths
# from start to dest node.

CreateGraph graph5

AddNode graph5 S
AddNode graph5 1
AddNode graph5 2
AddNode graph5 3
AddNode graph5 D

AddEdge graph5 S 1 1.0
AddEdge graph5 1 2 1.0
AddEdge graph5 2 S 1.7
AddEdge graph5 S 3 2.0
AddEdge graph5 3 D 3.0
AddEdge graph5 S D 2.0

FindPath graph5 S D
