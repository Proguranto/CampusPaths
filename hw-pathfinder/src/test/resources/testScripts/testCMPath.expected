# Find a path with Dijkstra's algorithm when
# the graph is cyclic and there is multiple paths
# from start to dest node.

created graph graph5

added node S to graph5
added node 1 to graph5
added node 2 to graph5
added node 3 to graph5
added node D to graph5

added edge 1.000 from S to 1 in graph5
added edge 1.000 from 1 to 2 in graph5
added edge 1.700 from 2 to S in graph5
added edge 2.000 from S to 3 in graph5
added edge 3.000 from 3 to D in graph5
added edge 2.000 from S to D in graph5

path from S to D:
S to D with weight 2.000
total cost: 2.000
